projects/ad7606x_fmc/common/ad7606x_fmc.txt

```diff
@@ -1,36 +1,47 @@
-FMC_pin   FMC_port       Schematic_name   System_top_name   IOSTANDARD    Termination
+FMC_pin   FMC_port         Schematic_name   System_top_name     IOSTANDARD    Termination
+  
```

Comment:
Remove line 2 as it will generate a file that starts with an empty line

Reply:
Done.